ledger/eval_simple_test.go

```diff
@@ -211,6 +212,359 @@ func TestBlockEvaluator(t *testing.T) {
 	require.Equal(t, bal2new.MicroAlgos.Raw, bal2.MicroAlgos.Raw-minFee.Raw)
 }
 
+// TestMiningFees ensures that the proper portion of tx fees go to the proposer
+func TestMiningFees(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	// Lots of balance checks that would be messed up by rewards
+	genBalances, addrs, _ := ledgertesting.NewTestGenesis(ledgertesting.TurnOffRewards)
```

Comment:
Are we 100% that rewards won't interfere in actual balance checks?

Reply:
By interfere here, I mean that the test checks balances to make sure they came out right.  If rewards could show up occasionally, it makes it much harder to write a test that says "X earned exactly this much".

I think you have a different concern in mind?

Reply:
I was trying to think of any downsides to disabling awards for this test, but it's probably fine.